packages/config/src/config/moonbase/assets/dev.ts

```diff
@@ -36,12 +39,32 @@ export const DEV: MoonbaseXcmConfig = {
           [XTokensExtrinsicCurrencyTypes.OtherReserve]: cloverDevId,
         }),
     },
+    [pioneer.key]: {
+      source: pioneer,
+      balance: balance.tokens({
+        [BalanceCurrencyTypes.FungibleToken]: pioneerDevId,
+      }),
+      sourceFeeBalance: balance.system(),
+      extrinsic: extrinsic
+        .xTokens()
+        .transfer()
+        .successEvent(XTokensExtrinsicSuccessEvent.TransferredMultiAssets)
+        .origin(pioneer)
+        .asset({ [XTokensExtrinsicCurrencyTypes.FungibleToken]: pioneerDevId }),
+    },
   },
   withdraw: {
     [clover.key]: withdraw.xTokens({
       balance: balance.assets(cloverDevId),
       destination: clover,
       feePerWeight: 50_000,
     }),
+    [pioneer.key]: withdraw.xTokens({
+      balance: balance.tokens({
```

Comment:
One thing that we could improve for the tokens balance we could change:
```typescript
balance: balance.tokens().fungibleToken(pioneerDevId),
```
and the same for other token types.

Reply:
I have applied this in the latest commit